devito/passes/clusters/cse.py

```diff
@@ -1,7 +1,7 @@
-from collections import Counter, OrderedDict
+from collections import Counter, OrderedDict, namedtuple
 from functools import singledispatch
 
-from sympy import Add, Function, Indexed, Mul, Pow
+from sympy import Add, Function, Indexed, Mul, Pow, Eq as Eqs
```

Comment:
I would just `import sympy` and then use `sympy.Eq`. The explicit way might be more verbose, but it makes the code easier to understand

Reply:
done